WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeSearchFragment.java

```diff
@@ -63,6 +65,8 @@ private void restoreState(Bundle savedInstanceState) {
     @Override
     public void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
+        outState.putString(BUNDLE_LAST_SEARCH, mLastSearch);
+        mSavedInstanceState = outState;
```

Comment:
I see `mSavedInstanceState` keeps a reference to `outState`. Is that really necessary? Can we remove `mSavedInstanceState` altogether?